packages/io/src/types/index.ts

```diff
@@ -1,31 +1,34 @@
 import type { CKBComponents } from '@ckb-lumos/rpc/lib/types/api'
-
+import { Key } from 'path-to-regexp'
 export interface Listener<T> {
   on(listen: (obj: T) => void): void
 }
 
-export type Middleware = (ctx: Context, next: () => Promise<void>) => Promise<void>
+export type JsonValue = null | number | string | { [key: string]: JsonValue } | JsonValue[]
```

Comment:
Could `{ [key: string]: JsonValue }` be expressed with `Record<string, JsonValue>`?

Reply:
use`Record<string, JsonValue> ` will have `Type alias 'JsonValue' looply references itself` error

Reply:
Got it